docs/resources/azurerm_key_vault.md.erb

```diff
@@ -0,0 +1,109 @@
+---
+title: About the azurerm_key_vault Resource
+platform: azure
+---
+
+# azurerm\_key\_vault
+
+Use the `azurerm_key_vault` InSpec audit resource to test properties and configuration of
+an Azure Key Vault.
+<br />
+
+## Availability
+
+### Installation
+
+This resource is available in the `inspec-azure` [resource
+pack](https://www.inspec.io/docs/reference/glossary/#resource-pack). To use it, add the
+following to your `inspec.yml` in your top-level profile:
+
+    depends:
+      inspec-azure:
+        git: https://github.com/inspec/inspec-azure.git
+
+You'll also need to setup your Azure credentials; see the resource pack
+[README](https://github.com/inspec/inspec-azure#inspec-for-azure).
+
+### Version
+
+This resource first became available in 1.1.0 of the inspec-azure resource pack.
+
+## Syntax
+
+The `resource_group` and `vault_name` must be given as a parameter.
+
+    describe azurerm_key_vault(resource_group: 'inspec-resource-group', vault_name: 'vault-101') do
+      it            { should exist }
+      its('name')   { should eq('vault-101') }    
+    end
+
+<br />
+
+## Examples
+
+If a Key Vault is referenced with a valid `Resource Group` and `Vault Name`
+
+    describe azurerm_key_vault(resource_group: 'my-rg', vault_name: 'vault-101') do
+      it { should exist }
+    end
+
+If a Key Vault is referenced with an invalid `Resource Group` or `Vault Name`
+
+    describe azurerm_key_vault(resource_group: 'invalid-rg', vault_name: 'vault-null') do
+      it { should_not exist }
+    end
+
+<br />
+
+## Parameters
+
+  - `resource_group` - The resource Group to which the Key Vault belongs.
+  - `vault_name` - The unique name of the Key Vault.
+
+
+## Properties
```

Comment:
I renamed `Properties` to `Attributes` since there's also a field called `properties`.